### PR TITLE
Fix the name of events overflowing onto the time

### LINKED ISSUE #6582: Calendar entry text may overflow with time
Please see the screenshot:
screenshot-appointments

The calendar entry title ("Meeting 1 with much more text here") overflows the time indicator.

## Test notes
- [ ] The event title does not overflow:
  - [ ] at various viewport sizes
  - [ ] when multiple events are stacked with the same start times
  - [ ] when the event duration is short such as 10 minutes

### PR DESCRIPTION
**Pull request by rezbyte**
I also experimented with a CSS flow based `CalendarEventBubble` to simplify the code but that is still too difficult because of the absolute positioning and sizing used in `CalendarDayEventsView`.

Closes #6582.

### CODE REVIEW

File: src/gui/main-styles.ts [hunk 1]
```diff
@@ -554,6 +554,13 @@ styles.registerStyle("main", () => {
 			"min-width": 0,
 			"white-space": "nowrap",
 		},
+		".text-ellipsis-multi-line": {
+			display: "-webkit-box",
```
Comment (charlag) [inline src/gui/main-styles.ts:558]: oh, today I've learned an obscure CSS trick, thank you!

perhaps we could add some info on it? I have found that it's mentioned here and despite being prefixed it is actually standardized and widely supported
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

**Review by charlag — APPROVED** [review]
I didn't test yet but looks good, thanks!

**Comment by marcosdiasalves** [comment]
Not fixed for me.
Tuta V220.240411.0.
Samsung Galaxy A30 - Android 11 - One UI 3.1

WhatsApp Image 2024-04-12 at 17 14 00

**Comment by marcosdiasalves** [comment]
Another comment: Why to you have "April 2024" in English and "Abril **_de_** 2024" in Brazilian Portuguese? You could remove this "de".

**Pull request merged**